include/rthw.h

```diff
@@ -74,6 +75,7 @@ void rt_hw_interrupt_init(void);
 void rt_hw_interrupt_mask(int vector);
 void rt_hw_interrupt_umask(int vector);
 rt_isr_handler_t rt_hw_interrupt_install(int              vector,
+                                         int              priority,
```

Comment:
Changes like this will break many BSPs relying on this API. Do you fix all of them?